packages/ui/src/domainModules/collectionMammals/components/MammalManager/ResultOptionsBar/index.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Button, Grid, Icon, Menu, Popup, Segment } from 'semantic-ui-react'
+import 'common/dist/semantic.css' // eslint-disable-line
+import 'react-rangeslider/lib/index.css'
+
+const propTypes = {
+  activeItem: PropTypes.string.isRequired,
+  onExportCSV: PropTypes.func.isRequired,
+  onFormTabClick: PropTypes.func.isRequired,
+  onSettingClick: PropTypes.func.isRequired,
+  onTableTabClick: PropTypes.func.isRequired,
+  tabContent: PropTypes.string.isRequired,
+}
+
+export class ResultOptionsBar extends Component {
+  render() {
+    const {
+      activeItem,
+      onExportCSV: handleExportToCSV,
+      onFormTabClick: handleFormTabClick,
+      onSettingClick: handleSettingClick,
+      onTableTabClick: handleTableTabClick,
+      tabContent,
+    } = this.props
+
+    return (
+      <div>
+        <Menu attached="top" tabular>
+          <Menu.Item
+            active={activeItem === 'form'}
+            name="form"
+            onClick={() => handleFormTabClick()}
+          >
+            <Icon name="wordpress forms" size="large" />
+          </Menu.Item>
+          <Menu.Item
+            active={activeItem === 'table'}
+            name="table"
+            onClick={() => handleTableTabClick()}
+          >
+            <Icon name="table" size="large" />
+          </Menu.Item>
+          <Menu.Menu position="right">
+            <Grid textAlign="center" verticalAlign="middle">
+              <Grid.Column>
+                <Popup
+                  content={
+                    <Button
+                      content="Export result to csv"
+                      onClick={() => handleExportToCSV()}
```

Comment:
We usually do camelCase also for abbreviations so it would be `handleExportToCsv`. But maybe the button content would say `CSV` instead of `csv`